stups-spring-oauth2-server/pom.xml

```diff
@@ -74,7 +74,7 @@
         </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-logging</artifactId>
+            <artifactId>spring-boot-starter-log4j</artifactId>
```

Comment:
why the "step back" to old log4j instead of log4j2 (or just leaving as is with default spring boot logging - logback)?